train_autodeeplab.py

```diff
@@ -45,32 +59,65 @@ def __init__(self, args):
         self.criterion = SegmentationLosses(weight=weight, cuda=args.cuda).build_loss(mode=args.loss_type)
 
         # Define network
-        model = AutoDeeplab (num_classes=self.nclass, num_layers=12, criterion=self.criterion, filter_multiplier=self.args.filter_multiplier,
-                             block_multiplier=self.args.block_multiplier, step=self.args.step)
+        model = AutoDeeplab (num_classes=self.nclass, num_layers=12, criterion=self.criterion, filter_multiplier=self.args.filter_multiplier)
         optimizer = torch.optim.SGD(
-                model.weight_parameters(),
+                model.parameters(),
```

Comment:
Ido, in the paper the weights and alpha betas are trained separately, and as of now the default class parameters() method contains the alphas and betas, best to leave it as it was before? or maybe I'm missing something?

Reply:
Hi,
i don't remember changing anything related to the optimizer's parameters. perhaps it's just "left-overs" from me not working on the most updated version of your code?

what i did change was removing the need for the Architect class, and bringing the whole Architect logic into the main script. the reason i did that is because i had to call amp.initialize on a single model and the two optimizers.

Reply:
Yeah, with that you did good. okay great!